index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="UTF-8">
+<head>
+    <title>Osama's Portfolio</title>
+    <link type="text/css" rel="stylesheet" href="normalize.css">
+    <link type="text/css" rel="stylesheet" href="styles.css">
+</head>
+
+<!--[if lte IE 9]><body class="ieLegacy"><![endif]-->
+<!--[if !IE]><body><![endif]-->
+
+<header class="container">
+    <nav>
+        <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="portfolio.html">Portfolio</a></li>
+        </ul>
+    </nav>
+
+    <h1 class="homePageTitle">Osama's Portfolio</h1>
+</header>
+
+<footer class="container">
+    <div class="contactIcon">
+        <span>Contact me</span>
+        <a href="mailto:o.tahboob2000@gmail.com?Subject=Portfolio%20Query"></a>
+        <img src="assets/contactWhite.png" alt="Email me icon">
+    </div>
+
+    <div class="contactIconSmall">
+        <span>Contact me</span>
```

Comment:
not displayed

Reply:
displays in IE9 as a replacement to the icon, since link to generate email does not work in IE9 with icon, so it becomes visible in IE9 and the icon disappears.